app/src/main/res/values/strings.xml

```diff
@@ -150,6 +150,11 @@
     <string name="tracks">Tracks</string>
     <string name="users">Users</string>
     <string name="events">Events</string>
+    <string name="music_songs">Music/Songs</string>
+    <string name="music_videos">Music/Videos</string>
+    <string name="music_albums">Music/Albums</string>
+    <string name="music_playlists">Music/Playlists</string>
+    <string name="music_artists">Music/Artists</string>
```

Comment:
Some of these options may confuse the user, for example:

- `Videos` and `Music/Videos`
- `Playlists` and `Music/Playlists`

Maybe another prefix should be used? _**YT Music**_ or even **_YouTube Music_**.

Reply:
I was already wondering if this would be the best way to label it (for both the users, and the translators).

YouTube Music returning different titles isn't really an issue imho.

Reply:
I suggest to do something like this. Not sure, how we can implement this, but it is the best things I can up with until we have a proper search filter dialog.

<img src="https://user-images.githubusercontent.com/17365767/77205060-5ba00680-6af4-11ea-91b8-ffa5b419ae97.png" height="400">